cmd/scollector/collectors/haproxy_unix.go

```diff
@@ -0,0 +1,518 @@
+package collectors
+
+import (
+	"encoding/csv"
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+)
+
+var trackedInstances = []*trackedInstance{}
+
+type trackedInstance struct {
+	Tier     string
+	URL      string
+	Username string
+	Password string
+}
+
+// HAProxy registers an HAProxy collector.
+func HAProxy(user, pwd string) {
```

Comment:
I like method names to be good verbs. This is a noun, and I am not sure what it does when I See it in main. Maybe `StartHAProxyCollector` or something along those lines.